jerry-core/include/jerryscript-debugger.h

```diff
@@ -17,6 +17,7 @@
 #define JERRYSCRIPT_DEBUGGER_H
 
 #include <stdbool.h>
+#include "jerryscript-port.h"
```

Comment:
This is definitely not good here.

Reply:
We can change the init() to take a void*, thus avoid pulling in jerryscript-port.h

Reply:
Or repeat the forward declaration?
Or move the forward declaration into an API header?
@zherczeg What's the best do you think?

Reply:
Or repeat the forward declaration?
Or move the forward declaration into an API header and include that?
@zolWhat's the best do you think?